.github/workflows/ci.yml

```
@@ -2,19 +2,16 @@ name: CI
 
 # limit ci building to pushes to master not to get twice the notification email
 # from github.
-#
-# if you want to test your branch on your fork, just replace the 'branches'
-# filter below for 'push' with the one for 'pull_request' temporarily and drop
-# the commit when creating the PR.
 on:
+  workflow_dispatch:
   push:
     branches:
       - 'master'
   pull_request:
     branches: '*'
 
 jobs:
-  build:
+  interledger:
```

```suggestion
  build:
```

Per repo configuration requirement.

---

I'll make another `build` job, which just depends on the `interledger*` jobs -- this will keep the semantics the same as previously?

---

Aah yes, makes sense. The easiest thing that comes to mind is a single step of `true`, but perhaps more true answer would be to have it wait for the interledger one, but can't remember how did those events work.

---

`needs` sounds about right, resolving.